app/controllers/util/DynamicFormUtils.scala

```diff
@@ -51,50 +54,49 @@ class DynamicFormUtils(request: Request[AnyContent], defaultFieldValues: List[Fo
           .copy(fieldErrors = RadioButtonGroupField.validate(selectedOption, dependencies, radioButtonGroupField))
 
       case textField: TextField =>
-        val text = fieldValue.head._2.head
+        val text = fieldValue.head.optionsSelected.head
```

Comment:
Do you think it's worth moving this variable out of the match statement? It's repeated a few times here.